contents/customers/4dayweek.md

```diff
@@ -0,0 +1,66 @@
+
+“Building in public is a great way to get validation from the community,” says Phil. “It’s especially important in the early days of launching a product, when you want to get feedback, encourage discussion and ensure you’re staying accountable to all your promises!”
+
+The fact that Phil builds in public also made him naturally curious about PostHog, which also prides itself on its [transparency and open source culture](/handbook/company/values). 
+
+“I found out about PostHog through a friend in a Telegram group initially,” says Phil. “And I just knew from the content, the documentation, and the transparency that this was a good fit for me. I immediately became convinced that I needed to start A/B testing things with PostHog.”
+
+### Increasing conversions with social proof and A/B testing
+
+Phil was familiar with the _concept_ of A/B testing, but had never tried actually running a test before. Thankfully, with a little help from ChatGPT and [PostHog’s documentation](/tutorials/categories/experimentation), he quickly got up to speed and began planning how to run A/B tests that could make 4DayWeek even more successful.  
+
+“Everything test has to begin with a hypothesis,” says Phil. “In my case, I wanted to test if social proofs would help increase the number of newsletter signups. My hypothesis was that users would be more likely to convert if they had an indication that the service was popular and trusted by lots of people.”
+
+​
+![PostHog 4dayweek experiment](../images/customers/4dayweek/4dayweek-before-after.png)
+<Caption>Tests in PostHog can have multiple variants, but Phil kept his simple and split traffic 50/50 between the original (left) and the test (right) versions</Caption>
+
+
+As soon as the test was running Phil started to see a clear winner — enough that, even though PostHog tracks statistical significance automatically, he was confident in selecting a winner prematurely. 
+
+“I made the call to end the experiment early and deploy the test version,” says Phil. “It took less that two weeks to get that result and the difference was enough to justify it. At no point did the original control version perform better than the variant. I set it live and have been getting more subscriptions ever since!” 
+
+<BorderWrapper>
+<Quote
+    imageSource="/images/customers/4dayweek_phil.jpg"
+    size="md"
+    name="Phil McParlane"
+    title="Founder, 4DayWeek"
+    quote={`“I started with testing, but then I started tracking events and building dashboards too. I’ve realized PostHog is something I’ve been looking for for a while — somewhere that I can have all the tools and analytics I need all in one place.”`}
+/>
+</BorderWrapper>
+
+### Even a negative result is a useful result
+
+Part of the value in running these experiments for Phil is that they help validate ideas and ward off blind optimism — a trap many other solo founders have fallen prey to. 
```

Comment:
```suggestion
Phil values running experiments because they help validate ideas and ward off blind optimism – a trap many solo founders fall prey to. 
```